soccer/gameplay/root_play.py

```diff
@@ -39,10 +39,18 @@ def execute_running(self):
         ################################################################################
 
         if main.game_state().is_stopped():
-            if not isinstance(self.play, plays.stopped.Stopped):
-                logging.info("Running 'Stopped' play due to game state change")
-                self.play = plays.stopped.Stopped()
-                self._currently_restarting = True
+            if main.game_state().is_placement():
```

Comment:
The refbox can only send a placement command if it is in the stopped state already.